packages/string-templates/src/helpers/list.js

```diff
@@ -3,6 +3,8 @@ const helperList = require("@budibase/handlebars-helpers")
 
 let helpers = undefined
 
+const helpersToRemove = ["sortBy"]
```

Comment:
How come we're removing `sortBy`?

Reply:
Our handlebars don't work fine with returned objects, and sort by only returns objects. This is the only one we have, and obviously nobody is using it. This is the current result:
<img width="309" alt="image" src="https://github.com/Budibase/budibase/assets/15987277/c6b4e6fe-15ac-45ba-bddd-96aa75a8efa0">


This kind of helpers are easily achievable via pure js, so easier to drop them